controller/grpc.go

```diff
@@ -938,6 +939,18 @@ func (g *grpcAPI) CreateDeployment(req *api.CreateDeploymentRequest, ds api.Cont
 	return maybeError(sub.Err)
 }
 
+func (g *grpcAPI) SetRoutes(ctx context.Context, req *api.SetRoutesRequest) (*api.SetRoutesResponse, error) {
+	changes, state, err := g.routeRepo.Set(req.AppRoutes, req.DryRun, req.ExpectedState)
+	if err != nil {
+		return nil, err
```

Comment:
Should this error be converted to a `grpc.Error`?

Reply:
Errors refactored in aa4ceb32.